README.rst

```diff
@@ -1,47 +1,66 @@
-FreeType high-level python API
-==============================
+FreeType (high-level Python API)
+================================
 
-Freetype python provides bindings for the FreeType library. Only the high-level API is bound.
+Freetype Python provides bindings for the FreeType library. Only the
+high-level API is bound.
 
 Documentation available at: http://freetype-py.readthedocs.org/en/latest/
 
 Installation
 ============
 
-To be able to use freetype python, you need the freetype library version 2
-installed on your system.
+**From PyPI, recommended**: `pip install freetype-py`. This will install the
```

Comment:
Note that on PyPI, you will also upload a sdist (.tar.gz or .zip) in addition to the precompiled wheel packages.
That will be used for architectures or python versions for which pip cannot find a suitable wheel, or if the user passes the --no-binary option. When installing from sdist, the value of the FREETYPEPY_BUNDLE_FT environment variable will determine whether the embedded shared lib will be built or not.